lib/bumper.js

```diff
@@ -98,6 +98,12 @@ class Bumper {
         return this._maybeBumpVersion(info, 'package.json')
       })
       .then((info) => {
+        if (fs.existsSync('package-lock.json')) {
```

Comment:
I'm not crazy about the use of a sync `fs` method here, when we're already doing things async with promises, but I suppose we already have sync reading of the json config file, so it's not the only example of using the sync version. It's fine for now, I'll think about refactoring it if/when I change the json read to async.

Reply:
I had originally planned to put this at the head where the package.json is read, but it made it difficult to toggle the existence...